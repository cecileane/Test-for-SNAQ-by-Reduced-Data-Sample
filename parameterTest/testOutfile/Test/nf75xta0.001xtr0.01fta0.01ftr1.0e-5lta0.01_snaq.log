optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.01,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 0.01.
rootname for files: nf75xta0.001xtr0.01fta0.01ftr1.0e-5lta0.01_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 61158
seed: 61158 for run 1
Sat Oct  1 17:38:07 2016

 BEGIN SNaQ for run 1, seed 61158 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 236 (failures: 75) with -loglik=165.4149 and ht_min=[0.33546,9.7588,0.46105,0.03806,0.86609]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 64		 64		 3	 |	 0		 0		 --
mvtarget	 71		 71		 1	 |	 0		 0		 --
chdir		 21		 21		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 79		 39		 2	 |	 0		 0		 --
Total		 236		 196		 8	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 165.41489611056576
((2,1):0.46105361246483695,(((5,#H7:::0.33546292649704357):9.758796736418834,6):0.8660940465505813,4):0.03806465353211488,(3)#H7:::0.6645370735029564);
---------------------
seed: 91814 for run 2
Sat Oct  1 17:38:16 2016

 BEGIN SNaQ for run 2, seed 91814 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 120 (failures: 75) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 30		 30		 3	 |	 0		 0		 --
mvtarget	 36		 36		 1	 |	 0		 0		 --
chdir		 15		 15		 3	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 37		 30		 1	 |	 0		 0		 --
Total		 120		 113		 9	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 2, -loglik of best 155.4962573461947
(4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7:::0.626975037501465):2.1558541373328457):2.1558541373328457,#H7:::0.3730249624985349);
---------------------

Sat Oct  1 17:38:21 2016
MaxNet is (4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7:::0.626975037501465):2.1558541373328457):2.1558541373328457,#H7:::0.3730249624985349); 
with -loglik 155.4962573461947
