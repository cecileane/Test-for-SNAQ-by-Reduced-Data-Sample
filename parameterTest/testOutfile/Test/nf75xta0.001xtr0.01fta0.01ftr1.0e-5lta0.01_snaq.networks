(4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7:::0.626975037501465):2.1558541373328457):2.1558541373328457,#H7:::0.3730249624985349);, with -loglik 155.4962573461947 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(1,((3,(6,5):0.319952435610033):0.0,(4)#H7:::0.6779646633461487):1.1211284467632396,(2,#H7:::0.32203533665385126):5.664047539499261);, with -loglik 158.69760508271563
(4,((3,(6,5):0.32834437264854915):0.0,#H7:::0.48213149886577134):0.5625389426358895,((1)#H7:::0.5178685011342287,2):5.461451879624049);, with -loglik 170.80024634279184
(4,((3,(6,5):0.36786882709947605):0.0)#H7:1.0017848039543442::0.7677650275348106,((1,#H7:0.7519309886372448::0.23223497246518943):5.274828565502837,2):1.4804680428445312);, with -loglik 175.08110457796406
