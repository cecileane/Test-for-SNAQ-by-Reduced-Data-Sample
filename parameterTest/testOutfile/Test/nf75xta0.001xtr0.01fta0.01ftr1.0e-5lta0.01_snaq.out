(4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7:::0.626975037501465):2.1558541373328457):2.1558541373328457,#H7:::0.3730249624985349); -Ploglik = 155.4962573461947
 Dendroscope: (4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7):2.1558541373328457):2.1558541373328457,#H7);
 Elapsed time: 14.516958888 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (4,((3,(6,5):0.3353461557625145):0.12573297163779387,(1,(2)#H7:::0.626975037501465):2.1558541373328457):2.1558541373328457,#H7:::0.3730249624985349);, with -loglik 155.4962573461947
 ((2,1):0.46105361246483695,(((5,#H7:::0.33546292649704357):9.758796736418834,6):0.8660940465505813,4):0.03806465353211488,(3)#H7:::0.6645370735029564);, with -loglik 165.41489611056576
-------
